model/src/main/java/org/projectnessie/api/params/RefLogParams.java

```diff
@@ -52,16 +52,30 @@
   @QueryParam("endHash")
   private String endHash;
 
+  @Parameter(
+      description =
+          "A Common Expression Language (CEL) expression. An intro to CEL can be found at https://github.com/google/cel-spec/blob/master/doc/intro.md.\n\n"
+              + "Usable variables within the expression are:\n\n"
+              + "- 'reflog' with fields 'refLogId' (string), 'refName' (string), 'commitHash' (string), 'parentRefLogId' "
+              + "(string), ',operation' (string), 'operationTime' (long)\n\n"
+              + "Hint: when filtering entries, you can determine whether entries are \"missing\" (filtered) by "
+              + "checking whether 'ReflogResponseEntry.parentRefLogId' is different from the hash of the previous "
+              + "reflog in the log response.")
+  @QueryParam("filter")
+  private String filter;
+
   public RefLogParams() {}
 
-  private RefLogParams(String startHash, String endHash, Integer maxRecords, String pageToken) {
+  private RefLogParams(
```

Comment:
The whole class is missing a `RefLogParamsTest`, similar to the other params test classes.

Reply:
Added it similar to `CommitLogParamsTest`